Gaijinizer patch/CommonEvents_789.txt

```diff
@@ -1,11 +1,11 @@
 ﻿> GAIJINIZER PATCH FILE
 > BEGIN STRING
 \\N[1]は、キメラビーストに敗北してしまった……
-> CONTEXT: CommonEvents_789/107/Dialogue < UNTRANSLATED
-> CONTEXT: CommonEvents_789/142/Dialogue < UNTRANSLATED
-> CONTEXT: Troops/362/pages/0/17/Dialogue < UNTRANSLATED
-> CONTEXT: Troops/387/pages/0/18/Dialogue < UNTRANSLATED
-
+> CONTEXT: CommonEvents_789/107/Dialogue
+> CONTEXT: CommonEvents_789/142/Dialogue
+> CONTEXT: Troops/362/pages/0/17/Dialogue
+> CONTEXT: Troops/387/pages/0/18/Dialogue
+\\N[1] has been defeated by Chimera Beast...#Ron
 > END STRING
```

Comment:
Feel like we should add "the" before "Chimera", think it still makes sense for the group battle too.